datasets/ONS-UK-trade-in-services-by-business-characteristics/spec.md

```diff
@@ -0,0 +1,50 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1671\cocoasubrtf600
```

Comment:
I'm not sure what's going on here, but all this stuff with backslashes isn't markdown and makes the document really hard to read.

Reply:
Done this.